### PR TITLE
Added pt-PT translations and improved/fixed README

### PR DESCRIPTION
**Pull request by jtsimoes**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # nodebb-plugin-calendar
 
-This plugin adds a site-wide calendar to NodeBB.
+This plugin adds a site-wide calendar to NodeBB. It also lets you create and customize events.
 
-## Installation
-Current alpha/beta version can be installed by doing the following:
+![Event](https://i.imgur.com/Rafqlcm.png)
```
Comment (pitaj) [inline readme.md]: Could you crop these in a little bit, so they're more mobile friendly?

File: readme.md [hunk 2]
```diff
@@ -31,7 +30,7 @@ Before do anything with the Calendar, make sure you have the correct settings se
  - Add **Reminders** determine when subscribers get notifications about the event.
  - **Mandatory** events disable responses and send all reminders to any user that can view the event.
 
-####In The Works
+## In The Works
```
[the remark this answers is not in the document]
Comment (jtsimoes) [reply readme.md]: Got it! So it was only the space missing

**Comment by jtsimoes** [comment]
@pitaj All the requested changes were made

**Comment by pitaj** [comment]
Thanks!

**Pull request merged**